dltxio/v2.getpaidinbitcoin.com.au
# GPIB-249

changed filed name from payroll contact to contact name
added domain name check on the backend https://github.com/dltxio/gpib.admin.api/pull/167, really need marge the backend pr first.
To test,
1. run the app and open onboarding page (http://localhost:5000/enterprise), 
2. give the domain name which already exist like (gmail.com), expect error message showup when submit the application.
